components/issue-list-item.tsx

```diff
@@ -30,90 +27,107 @@ export default function IssueListItem({
+        {(issue?.fundingAmount > 0) && (
+          <>
+            <div className="p-0 col-md-6 mt-1">
+              <div className="bg-dark-gray w-100 issue-funding-progress">
+                <div
+                  className={`${
+                    percentage !== 100 ? "bg-primary" : "bg-success"
+                  } issue-funding-progress`}
+                  style={{ width: `${percentage}%` }}
+                />
+              </div>
+            </div>
+            <div
+              className={`issue-percentage-text caption-small py-0 pe-0 ps-1 pb-1 col-md-2 ${
+                percentage !== 100 ? "text-white" : "text-success"
+              }`}
+            >
+              {percentage.toFixed(0)}%
+            </div>
+          </>
+        )}
       </div>
     );
   }
 
-  function renderReviews() {
+  function createIssueDate(date: Date) {
     return (
-      <div className="d-flex align-items-center">
-        <span className="caption-small mr-1 text-white">
-          {(issue !== null && handleReviewsPr(issue?.pullRequests)) || 0}
-        </span>
-        <span className="caption-small text-white-40 text-uppercase">
-          {t("info.reviews", { count: handleReviewsPr(issue?.pullRequests)})}
-        </span>
-      </div>
+      <span className="caption-medium text-white-40">{formatDate(date)}</span>
     );
   }
 
-  function renderIssueData(state: IssueState) {
-    function handleFirstChildren() {
-      if (state?.toLowerCase() === "ready") {
-        return renderProposals();
-      } else if (state?.toLowerCase() === "closed") {
-        return renderReviews();
-      } else {
-        return (
-          <div className="d-flex align-items-center">
-            <span className="caption-small mr-1 text-white">
-              {(issue !== null && issue.working?.length) || 0}
-            </span>
-            <span className="caption-small text-white-40 text-uppercase">
-              {t("info.working")}
-            </span>
-          </div>
-        );
-      }
-    }
-
-    if (!["draft", "pending", "canceled"].includes(state?.toLowerCase())) {
-      return (
-        <div className="d-flex align-center flex-wrap align-items-center justify-content-md-start mt-2 gap-20">
-          {handleFirstChildren()}
-          <div className="d-flex align-items-center">
-            <span className="caption-small mr-1 text-white">
-              {(issue !== null && issue.pullRequests?.length) || 0}
-            </span>
-            <span className="caption-small text-white-40 text-uppercase">
-              {t("info.pull-requests", { count: issue?.pullRequests?.length})}
-            </span>
-          </div>
-          {state?.toLowerCase() === "ready"
-            ? renderReviews()
-            : renderProposals()}
-          {state?.toLowerCase() !== "draft" && (
-            <DateLabel date={issue?.createdAt} />
-          )}
-        </div>
-      );
-    } else {
-      return;
-    }
+  function handleIssueState() {
```

Comment:
repeated code -- we should be able to abstract this function into a helper and provide it with everything it needs to discern the issueType so we don't have to mantaign more than one "handleIssueState" function [which should really be called "getIssueState"]